src/admin/index.ts

```diff
@@ -61,6 +61,7 @@ export const generateAdminJSConfig = () => ({
   rootPath: '/admin',
   branding: {
     companyName: 'AdminJS demo page',
+    favicon: '/favicon.ico',
```

Comment:
[OPT] I think that nowadays .png files great works with favicon and are more popular :)

Reply:
We didn't have a favicon file at all in this repository and I just copied one from our landing page. Not sure if I can just change the extension or I'd have to do something extra.